dev/TeachingTip/InteractionTests/TeachingTipTests.cs

```diff
@@ -448,6 +448,7 @@ public void TipsWhichDoNotFitDoNotOpen()
                     UseTestWindowBounds(10, 10, 10, 10);
 
                     elements.GetShowButton().Invoke();
+                    Wait.ForIdle();
```

Comment:
We have an extension method InvokeAndWait() that you can call that does this in one line.

Reply:
Gave it a run - strangely this actually breaks the test case (outside of debug mode at least) despite the fact that the Wait.ForIdle() statement inside InvokeAndWait is executed correctly. 

Irrespective of which method is used there is a suspicion of some level of unreliability regarding this test case.

Reply:
How does the test case fail? Some of these I left disabled because there seemed to be a logic error in the product (the test was expecting the right result but the product code wasn't producing it). Maybe the test is catching a bug now.

Reply:
Hard to say - after adding a wait to every invoke I no longer can replicate the same failures that show up in the tests run in the pipeline ("Works on my machine 🤷‍♂ "). Every test was re-enabled except for autoPlacement because it has completely separate problems. It's possible there's another issue.